packages/components/addon/components/hds/table/index.hbs

```diff
@@ -1,16 +1,11 @@
 <table class="hds-table {{if @isStriped 'hds-table--striped'}}" data-test-table ...attributes>
   <thead class="hds-table__thead" data-test-table-head>
-    <Hds::Table::Row>
-      {{yield
-        (hash SortBy=(component "hds/table/sort-by" currentField=@sortProperty sortDescending=this.sortDescending))
-        to="head"
-      }}
-    </Hds::Table::Row>
+    {{yield to="head"}}
   </thead>
   <tbody class="hds-table__tbody" data-test-table-body>
     {{#if @model}}
-      {{#each @model as |row|}}
-        {{yield row to="body"}}
+      {{#each @model as |modelItemData|}}
```

Comment:
I have noticed that in the Ember ecosystem, a "row" of data in the model is called `record`. Do you think it would make it cleared to call this entity `record` instead of `modelItemData` (which is a bit of a mouthful)?